.ci/appveyor.bat

```diff
@@ -0,0 +1,56 @@
+@echo off
+
+::----------------------------------------------------------------------
+:: Appveyor bat file to run on local for windows users.
+:: Example of usage
+:: ./.ci/appveyor.bat  verify_without_checkstyle
+::----------------------------------------------------------------------
+
+SET OPTION=%1
+goto :%OPTION% 2>nul
+IF ERRORLEVEL 1 CALL :DEFAULT_CASE
+ECHO Done.
+EXIT
+
+:sevntu
+mvn -e verify -DskipTests -DskipITs -Dpmd.skip=true^
+ -Dspotbugs.skip=true -Djacoco.skip=true -Dxml.skip=true
+    goto END_CASE
+
+:verify_without_checkstyle
+mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true
+  goto END_CASE
+
+:verify_without_checkstyle_JDK11
+mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true
+  goto END_CASE
+
+:verify_without_checkstyle_JDK12
+mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true^
+ -Dtest=!FileContentsTest#testGetJavadocBefore,!FileTextTest#testFindLine*,^
+!MainFrameModelPowerTest#testOpenFileWithUnknownParseMode,^
+!TokenUtilTest#testTokenValueIncorrect2,^
+!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose^
+ -Djacoco.skip=true
+  goto END_CASE
+
+:site_without_verify
+mvn -e -Pno-validations site
+  goto END_CASE
+
+:site_without_verify_jdk12
+mvn -e -Pno-validations site^
+ -Dtest=!FileContentsTest#testGetJavadocBefore,!FileTextTest#testFindLine*,^
+!MainFrameModelPowerTest#testOpenFileWithUnknownParseMode,^
+!TokenUtilTest#testTokenValueIncorrect2,^
+!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose^
+ -Djacoco.skip=true
+  goto END_CASE
+
+:DEFAULT_CASE
+  echo unknown option

```

Comment:
Can we print unexpected option like in travis.sh?

Reply:
Yes, i added that